wmfdata/spark.py

```diff
@@ -143,26 +167,9 @@ def get_custom_session(
         elif os.path.isfile(os.path.join(f"/usr/bin/python{python_version()}")):
             os.environ["PYSPARK_PYTHON"] = f"/usr/bin/python{python_version()}"
 
-        if "PYSPARK_PYTHON" in os.environ:
```

Comment:
Why remove this message?  Things can get pretty weird if an unexpected python is used.

Reply:
PySpark prints essentially the same message already when starting a session: for example, `PYSPARK_PYTHON=/usr/lib/anaconda-wmf/bin/python3`.